File: api/src/main/java/net/onelitefeather/pandorascluster/dbo/player/LandMemberDBO.java
package net.onelitefeather.pandorascluster.dbo.player;

import net.onelitefeather.pandorascluster.api.enums.LandRole;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public interface LandMemberDBO {

    @Nullable
    Long id();

    @NotNull
    LandPlayerDBO member();

    @NotNull
    LandRole role();
}

File: api/src/main/kotlin/net/onelitefeather/pandorascluster/dbo/player/LandMemberDBO.kt
package net.onelitefeather.pandorascluster.dbo.player

import net.onelitefeather.pandorascluster.api.enums.LandRole

interface LandMemberDBO {

    fun id(): Long?

    fun member(): LandPlayerDBO

    fun role(): LandRole
}